### PR TITLE
ft(S3C-3003): Add repair task and macros

### PR DESCRIPTION
**Pull request by tmacro**

### CODE REVIEW

**Review by rahulreddy — APPROVED** [review]

File: libV2/tasks/Repair.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+const BaseTask = require('./BaseTask');
+const config = require('../config');
+const { LoggerContext } = require('..//utils');
```
Comment (ilkescality) [inline libV2/tasks/Repair.js]: not sure if `//` intentional but just pointing out just in case

**Review by ilkescality — APPROVED** [review]

**Pull request merged**